### PR TITLE
corrected error msg

### PR DESCRIPTION
**Pull request by upadhyay-prashant**
BUG-ID: https://jira.ril.com/browse/JCC-30

### CODE REVIEW

**Comment by upadhyay-prashant** [comment]
@varunarya10 , @anantkaushik89   please review this. BUG ID is listed above.

File: nova/exception.py [hunk 1]
```diff
@@ -1244,6 +1244,9 @@ class FlavorMemoryTooSmall(NovaException):
 class FlavorDiskTooSmall(NovaException):
     msg_fmt = _("Flavor's disk is too small for requested image.")
 
+class BlockDeviceSizeTooSmall(NovaException):
+    msg_fmt = _("Root Block device is too small for requested image.%(reason)s")
```
Comment (anantkaushik89) [inline nova/exception.py:1248]: Can you check whether the intended reason is being displayed in place of %(reason)s ?

**Comment by anantkaushik89** [comment]
Looks fine to me other than the small comment mentioned.

**Comment by upadhyay-prashant** [comment]
if a min disk for an image is 2 GB, and we try to boot it with root volume 1 GB following msg is produced. 

nova boot --flavor 1 --block-device source=image,id=9dd75c68-d8b7-424a-879c-bb43c83706b9,dest=volume,size=1,shutdown=remove,bootindex=0  --nic net-id=5f2dd98d-2ab4-432d-9248-5fc961c997d1 pra1
ERROR (BadRequest): Root Block device is too small for requested image. Minimum size required for this image is 2 GB. (HTTP 400) (Request-ID: req-8bc66bce-0776-46e7-b7d3-32360052de00)

So yes, Intended msg is printed.

**Pull request merged**